index2.html

```diff
@@ -0,0 +1,37 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+   
+<!-- The core Firebase JS SDK is always required and must be listed first -->
+<script src="https://www.gstatic.com/firebasejs/6.6.0/firebase-app.js"></script>
+<script src="https://www.gstatic.com/firebasejs/6.6.0/firebase-firestore.js"></script>
+
+    <title>CMS</title>
+    
+</head>
+<body>
+    <form id="add-coupon">
+    <input type="text" name="qrtext" placeholder="Event Name"/>
+    <input type="text" name="qremail" placeholder="Email Id"/>
+    <button class="button"> SUBMIT </button>
+    </form>
+
+<script>
+  // Your web app's Firebase configuration
+  var firebaseConfig = {
+    apiKey: "AIzaSyDyyVGG99b8wZfnZwK0NDczG-vXLPfSKss",
+    authDomain: "cms-api-b9a6f.firebaseapp.com",

```

Comment:
### [Security Vulnerability]
Again same thing!

Never expose your `API` keys or tokens.
`apiKey: "XXXXXX"`

Remove the API key, @muskantewari .